### PR TITLE
CORS-3741: [Nutanix] support multi-subnets in Machine providerSpec

### PR DESCRIPTION
**Pull request by yanhua121**
[CORS-3741](https://issues.redhat.com//browse/CORS-3741)
Nutanix: support multi-subnets in Machine providerSpec

Depends on https://github.com/openshift/api/pull/2077

### CODE REVIEW

**Review by elmiko — COMMENTED** [review]
this makes sense to me, i just have a minor suggestion

File: pkg/actuators/machine/utils.go [hunk 1]
```diff
@@ -23,6 +25,9 @@ const (
 	upstreamMachineClusterIDLabel = "sigs.k8s.io/cluster-api-cluster"
 	globalInfrastuctureName       = "cluster"
 	openshiftConfigNamespace      = "openshift-config"
+
+	taskTimeout = 5 * time.Minute  // 5m0s
+	sleepTime   = 10 * time.Second // 10s
```
[the remark this answers is not in the document]
Comment (yanhua121) [reply pkg/actuators/machine/utils.go]: changed as suggested.

**Review by elmiko — COMMENTED** [review]
/lgtm
/approve

it looked like the aws test failures were related to flakes on clone, i don't think it's related to this change.

**Pull request merged**